### DISCUSSION TITLE
Global `providerConfigRef` in Composition

### DISCUSSION THREAD

**Discussion by ytsarev** [crossplane/crossplane#2233] [Q&A] [answered]
Is there any way to specify `providerConfigRef` for all Composed resources within a Composition?

The obvious motivation is to avoid patching of `providerConfigRef` for each base Composed resource

**Comment by muvaf** [comment, marked as answer]
You can use `patchSets` to define a patch and refer to it by name in the patches https://crossplane.io/docs/v1.1/concepts/composition.html#specify-how-your-resource-may-be-composed

**Reply by ytsarev** [reply]
Yeah, that's exactly what I did. Just to double check: currently there is no way around repeating patchSet declaration for each base resource, correct?

**Reply by muvaf** [reply]
No, you have to call by its name.
